apps/frontend/src/app/2_molecules/DebugContent.tsx

```diff
@@ -76,15 +81,79 @@ export const DebugContent = () => {
       {
         title: `Approve ${symbol} tokens`,
         subtitle: `Allow Sovryn protocol to use ${symbol} tokens for the trade`,
-        contract: xusd,
-        fnName: APPROVAL_FUNCTION,
-        args: [address, parseUnits('2')],
+        request: {
+          type: TxType.signTransaction,
+          contract: xusd,
+          fnName: APPROVAL_FUNCTION,
+          args: [address, parseUnits('2')],
+        },
       },
       {
         title: `Transfer 2 ${symbol} tokens`,
-        contract: xusd,
-        fnName: 'transfer',
-        args: [wallets[0]?.accounts[0]?.address, parseUnits('2')],
+        request: {
+          type: TxType.signTransaction,
+          contract: xusd,
+          fnName: 'transfer',
+          args: [wallets[0]?.accounts[0]?.address, parseUnits('2')],
```

Comment:
```suggestion
          args: [account, parseUnits('2')],
```
Use `account` from `useAccount` hook instead.